### DISCUSSION TITLE
Avalonia Jobs

### DISCUSSION THREAD

**Discussion by mysteryx93** [AvaloniaUI/Avalonia#13402] [General]
I'm currently looking for a job; with 20+ years in .NET as a Senior Analyst Programmer & System Architect.

My ideal job would be Avalonia. I wrote [MvvmDialogs for Avalonia](https://github.com/mysteryx93/HanumanInstitute.MvvmDialogs) which IMO is a requirement for any Avalonia app.

I'm noticing though: on job websites, Avalonia is not even in the skillsets and no company looks for that! Companies don't even know that .NET is an option for cross-platform development like this.

So I was wondering, are there companies around here looking for .NET / Avalonia Senior developers?

**Comment by thevortexcloud** [comment]
Avalonia is still fairly small compared to WPF/what ever Microsoft is pushing these days. As such it's probably fairly rare you will find companies that are looking for Avalonia specifically. You will however probably find lots of companies that want WPF developers who can help modernise their code base/rewrite it to be cross platform who likely have never heard of Avalonia. Most of them though will probably just end up with a web app though if nobody tells them there are other options.

> Companies don't even know that .NET is an option for cross-platform development like this.

That's because Microsoft has for the most part gone out of their way to ignore anything that is not Windows (and sometimes Mac) when it comes to non web UI frameworks.

**Comment by mysteryx93** [comment]
I must say; this can be a deterrent for companies to adopt Avalonia.

There are very few companies using it, thus no job market for it.

And if a company uses it, finding developers with needed expertise will be just as difficult.

**Reply by thevortexcloud** [reply]
Considering how close Avalonia is to WPF in terms of how you program for it, most WPF devs can probably learn to use it in a day or two. It's really just a matter of starting discussions internally to get them using Avalonia. Some companies might be wary of using such a little known product though.

But there are some fairly big companies that are using it. Mainly Unity for their SCM, Jetbrains for their crossplatform C# tools and Github for their credential manager (where it's very likely they went with Avalonia instead of MAUI because Avalonia actually supports Linux despite being owned by Microsoft).

https://avaloniaui.net/Showcase
